src/lib/model.ts

```diff
@@ -118,27 +118,29 @@ export class Course {
 				if (module.events[0]) {
```

Comment:
what if there are more than 1 event?

Reply:
I think it will not be a problem as this checks if there is first event then it means that events array is not empty and we can look through it. Shall I change it to the check null and the check length?

Reply:
after having a chat with Kamesh, I've changed the if statement and pushed again